static/js/ReaderPanel.jsx

```diff
@@ -1258,7 +1258,7 @@ class ReaderDisplayOptionsMenu extends Component {
           settings={this.props.settings} />);
 
     var layoutOptions = [
-      {name: "continuous", fa: "align-justify", role: "radio", ariaLabel: "Show Text as a paragram" },
+      {name: "continuous", fa: "align-justify", role: "radio", ariaLabel: "Show Text as a paragram (?! what ??)" },
```

Comment:
This is the thing I called out as being pretty odd/needing a look. Obviously it shouldn't be merged as-is.